wp-content/themes/core/components/card/css/card.pcss

```diff
@@ -147,6 +189,18 @@
 			text-decoration: underline;
 		}
 	}
+
+	/* CASE: Style: Search */
+	.c-card--style-search & {
+		@mixin t-input;
+		margin-bottom: var(--spacer-10);
+		font-weight: 700;
```

Comment:
While this does match the designs, let's adjust it to `@mixin t-display-xx-small;` to keep it consistent with other "display" or heading styles.